### PR TITLE
Document behavior marshalling and streamlined translation saving.

### PR DESCRIPTION
**Pull request by markstory**
Refs cakephp/cakephp#9225

### CODE REVIEW

File: en/orm/behaviors.rst [hunk 1]
```diff
@@ -242,6 +242,29 @@ rename/remove finder methods when adding a behavior to a table. For example::
         ]
     ]);
 
+Defining Custom Marshalling Behavior Logic
+==========================================
+
+Behaviors can participate in defining how the custom fields they provide are
+marshalled by implementing the ``Cake\ORM\PropertyMarshalInterface``. This
+interface requires a single method be implemented::
```
Comment (antograssiot) [inline en/orm/behaviors.rst]: a single method *to* be implented ?

**Comment by antograssiot** [comment]
Nice job

File: en/orm/behaviors.rst [hunk 2]
```diff
@@ -242,6 +242,29 @@ rename/remove finder methods when adding a behavior to a table. For example::
         ]
     ]);
 
+Defining Custom Marshalling Behavior Logic
+==========================================
```
Comment (lorenzo) [inline en/orm/behaviors.rst]: I'd name this `Transforming request data into Entity properties` for easier search

**Comment by lorenzo** [comment]
There used to be some documentation explaining how to save translated records, but cannot remember where it is... should be remove/update that?

**Comment by markstory** [comment]
@lorenzo It is directly above the new section.

File: en/orm/behaviors.rst (hunk 1 above)
Comment (markstory) [inline en/orm/behaviors.rst]: Thank you.

File: en/orm/behaviors.rst (hunk 2 above)
Comment (markstory) [inline en/orm/behaviors.rst]: Good idea.

**Comment by markstory** [comment]
@antograssiot @lorenzo Updated.

**Pull request merged**